schema/types/isolator_linux_specific.go

```diff
@@ -227,7 +227,7 @@ func (l linuxSeccompBase) AssertValid() error {
 		return nil
 	}
 	for _, c := range l.val.Errno {
-		if !unicode.IsUpper(c) {
+		if !(unicode.IsUpper(c) || unicode.IsNumber(c)) {
 			return errors.New("errno must be an upper case string")
```

Comment:
This error message could be also updated there...
E.g. "errno must be a string containing only digits and upper case characters"

Reply:
As we wish, I think the additional OR-condition doesn't change the spirit of the previous error label. @jellonek's suggestion is strictly speaking more correct.